Update gsheets.md (#5451)
fishtown-analytics/docs.getdbt.com: website/docs/docs/use-dbt-semantic-layer/gsheets.md

- If you're using this extension, make sure you're signed into Chrome with the same Google profile you used to set up the Add-On. Log in with one Google profile at a time as using multiple Google profiles at once might cause issues.

## Query Builder functions

The Google Sheets **Query Builder** custom menu has the following capabilities:

| Menu items    | Description                                           |
|---------------|-------------------------------------------------------|
| Metrics       | Search and select metrics.                             |
| Group By      | Search and select dimensions or entities to group by. Dimensions are grouped by the entity of the semantic model they come from. You may choose dimensions on their own without metrics. |
| Time Range    | Quickly select time ranges to look at the data, which applies to the main time series for the metrics (metric time), or do more advanced filter using the "Custom" selection|
| Where         | Filter your data. This includes categorical and time filters. |
| Order By      | Return your data order.                              |
| Limit         | Set a limit for the rows of your output.               |

> Note: All metrics and dimensions have an info button which you can click into to get the description of the object as defined in your dbt project.

Reviewer: ```suggestion
Note: Click the info button next to any metric or dimension to see its defined description from your dbt project.
```

> **Modifying Time Granularity** 

Reviewer: ```suggestion
#### Modifying time granularity
```

> When selecting time dimensions in the group by menu, you will see a list of available time granularities below the selection. By default, the lowest granularity is selected. Metric Time is the default time dimension that you can group all your metrics by.

Reviewer: ```suggestion
When selecting time dimensions in the group by menu, you will see a list of available time granularities below the selection. By default, the lowest granularity is selected. Metric time is the default time dimension that you can group all your metrics by.
```

> When selecting time dimensions in the group by menu, you will see a list of available time granularities below the selection. By default, the lowest granularity is selected. Metric time is the default time dimension that you can group all your metrics by.

Reviewer: ```suggestion
When you select time dimensions in the **Group By** menu, you'll see a list of available time granularities. The lowest granularity is selected by default. Metric time is the default time dimension for grouping your metrics.
```
